src/main/window-manager.ts

```diff
@@ -81,7 +85,7 @@ export class WindowManager extends Singleton {
         show: false,
         minWidth: 700,  // accommodate 800 x 600 display minimum
         minHeight: 500, // accommodate 800 x 600 display minimum
-        titleBarStyle: "hiddenInset",
+        titleBarStyle: isWindows ? "hidden" : "hiddenInset",
```

Comment:
What about linux?

Reply:
No change, like on mac?

![Screen Shot 2021-12-08 at 11 41 10 AM](https://user-images.githubusercontent.com/40840436/145247987-b791f277-721a-4678-924d-618756282ae5.png)

Reply:
I guess I meant, why not on linux

Reply:
Yes, meant as a question to @aleksfront, if this is how it should look, wrt this PR

Reply:
ah, in an earlier DM from Alex he indicated to me this:
"Linux Lens window should have standard title bar with minimize/maximize/close, but menu should go under the sandwich"
so I guess there's a problem